src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Error {
```

Comment:
Never extend "Error" because it is only for problems with JVM.
Better to extend "Exception" here